src/pages/LandingPage.tsx

```diff
@@ -23,6 +23,12 @@ export const LandingPage: React.FC = () => {
                 <strong>Millions </strong>
                 <div className={styles.UnderlineBlock} /> of deaths from COVID-19 are underreported.
               </div>
+              <div className={styles.LabDescription}>
+                <span>Millions of deaths from COVID-19 are</span>
+              </div>
+              <div className={styles.LabDescription}>
+                <span>underreported. Millions of deaths from COVID-19.</span>
+              </div>
```

Comment:
Is there a reason why this is in two divs? If not, please move into a single div/span. Also, please place content in the content.ts file.